packages/cli/src/utils/governance.ts

```diff
@@ -0,0 +1,25 @@
+import { CeloTxPending } from '@celo/connect'
+import { ContractKit } from '@celo/contractkit'
+
+export async function checkProposal(
+  proposal: Array<Pick<CeloTxPending, 'to' | 'input' | 'value'>>,
+  kit: ContractKit
+) {
+  const governance = await kit.contracts.getGovernance()
+
+  for (const tx of proposal) {
+    console.log(tx)
+    if (!tx.to) {
+      continue
+    }
+
+    console.log(
+      await kit.web3.eth.call({
```

Comment:
can we reuse this function for the test-proposal `send` in addition to the check `call`s?